README.md

```diff
@@ -1,30 +1,28 @@
 Classnames
 ===========
 
-[![Version](http://img.shields.io/npm/v/classnames.svg)](https://www.npmjs.org/package/classnames)
-[![Build Status](https://travis-ci.org/JedWatson/classnames.svg?branch=master)](https://travis-ci.org/JedWatson/classnames)
-[![Supported by Thinkmill](https://thinkmill.github.io/badge/heart.svg)](http://thinkmill.com.au/?utm_source=github&utm_medium=badge&utm_campaign=classnames)
+[![NPM version](https://badgen.net/npm/v/classnames)](https://www.npmjs.com/package/classnames)
+[![Build status](https://badgen.net/travis/JedWatson/classnames)](https://travis-ci.org/JedWatson/classnames)
+[![NPM Weekly Downloads](https://badgen.net/npm/dw/classnames)](https://www.npmjs.com/package/classnames)
+[![License](https://badgen.net/npm/license/classnames)](https://www.npmjs.com/package/classnames)
+[![Supported by Thinkmill](https://thinkmill.github.io/badge/heart.svg)](https://thinkmill.com.au/?utm_source=github&utm_medium=badge&utm_campaign=classnames)
 
 A simple JavaScript utility for conditionally joining classNames together.
 
 Install with [npm](https://www.npmjs.com/), [Bower](https://bower.io/), or [Yarn](https://yarnpkg.com/):
 
-npm:
-```sh
+```bash
+# via NPM
```

Comment:
Should we preserve the casing of `npm`, `Bower`, and `Yarn`?

Reply:
Yikes written as I usually do, but it differs.
Shold be npm, Bower and yarn.
I'll fix it a bit later.

Reply:
Fixed.